product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -203,22 +204,21 @@ When running the first or second command to stop the Postgres instances, you may
 pg_ctl: PID file "/opt/postgres/data/postmaster.pid" does not exist
 Is server running?
 ```
-This error is expected when both instances of Postgres are already shut down and, therefore, does not prevent the upgrade from progressing.
+This error is expected when both instances of Postgres are already shut down. Therefore, it doesn't prevent the upgrade from progressing.
 !!!
 
 ### Check that you can upgrade safely
 
-The `bdr_pg_upgrade` tool has a `--check` option, which dry runs some of the upgrade process. We can use this option to ensure the upgrade will go smoothly. 
-
-However, first, we need a directory for the files created by `bdr_pg_upgrade`, which we can give ownership of to the user `postgres`. 
+The `bdr_pg_upgrade` tool has a `--check` option, which performs a dry run of some of the upgrade process. You can use this option to ensure the upgrade goes smoothly.
 
-Accordingly, create an `/upgrade` directory in the `/home` directory and then give ownership of that directory to the user `postgres`:
+However, first you need a directory for the files created by `bdr_pg_upgrade`. For this example, create an `/upgrade` directory in the `/home` directory. Then give ownership of the directory to the user postgres. 
```

Comment:
I'm not a fan of the word "accordingly." It sounds academic and it raises the reading level.